pages/docs/observability/sdk/python/instrumentation.mdx

```diff
@@ -364,6 +364,137 @@ with langfuse.start_as_current_span(name="initial-operation") as span:
         langfuse.update_current_trace(output={"final_status": "success"}, public=True)
 ```
 
+<Callout type="info">
+**Note:** For `user_id`, `session_id`, `metadata`, and `version`, consider using `propagate_attributes()` (see below) to ensure these attributes are applied to **all spans**, not just the trace object.
+</Callout>
+
+### Propagating Trace Attributes
+
+For certain trace attributes (`user_id`, `session_id`, `metadata`, `version`), you may want them to be automatically applied to **all spans** created within a trace, not just the trace object itself. This is important because Langfuse aggregation queries (e.g., filtering by user_id, calculating costs by session_id) will soon operate on individual spans/observations.
```

Comment:
Shall we call them trace attributes or attributes (no trace prefix)?

I think the sentence regarding soon is a bit unspecific without context but ok if we keep this here briefly before updating all of it when we are in a new consisten state